ru/turing/courses/lesson5/Main.java

```diff
@@ -1,7 +1,67 @@
 package ru.turing.courses.lesson5;
 
+import java.util.*;
+import static ru.turing.courses.lesson5.Student.bubbleSort;
+
 public class Main {
     public static void main(String[] args) {
-        //todo вставить сюда сортировки пузырьком, сделать для каждой свой метод
+        System.out.println("Пример сортировки пузырьком для массива int[]");
+        //объявляем и инициализируем массив
+        int[] intArray = {1, 5, 4, 3, 7};
+        System.out.println("Исходный массив: "+Arrays.toString(intArray));
```

Comment:
```suggestion
        System.out.println("Исходный массив: " + Arrays.toString(intArray));
```
Ctrl + Alt + L в IDEA для автоматического выравнивания всего и вся